server/tests/controller/article.spec.js

```diff
@@ -143,4 +164,46 @@ describe('Get articles', () => {
     await getArticles(req, res);
     expect(res.status).to.have.been.calledWith(500);
   });
+
+  it('should get a single article by Id', async () => {
```

Comment:
`await` is not used here, you should consider removing the `async` and use the `done`